e2-studio-star-rx72n-firmware/tests/test_rx_stack_monitor.c

```
@@ -0,0 +1,406 @@
+ * @see rx_stack_monitor_get_free_bytes()
+ * @since Version 1.0.0
+ */
+static void test_get_free_bytes_null_thread(void)
+{
+    uint32_t free_bytes = 0;
+
+    rx_err_t err = rx_stack_monitor_get_free_bytes(NULL, &free_bytes);
+
+    TEST_ASSERT_EQUAL(k_rx_err_null_ptr, err);
+    /* free_bytes must not have been written */
+    TEST_ASSERT_EQUAL(0u, free_bytes);
```

Clang (14.0.6)

[warning] integer literal has suffix 'u', which is not uppercase

(readability-uppercase-literal-suffix)